highcharts-angular/src/lib/highcharts-chart.directive.ts

```diff
@@ -0,0 +1,127 @@
+import {
+  Directive,
+  DestroyRef,
+  afterRenderEffect,
+  ElementRef,
+  inject,
+  input,
+  linkedSignal,
+  model,
+  output,
+  OutputEmitterRef,
+  untracked,
+  computed
+} from '@angular/core';
+import { toSignal } from '@angular/core/rxjs-interop';
+import { HighchartsChartService } from './highcharts-chart.service';
+import { HIGHCHARTS_CONFIG } from './highcharts-chart.token';
+import { Chart } from './types';
+
+
+@Directive({
+  selector: '[highcharts-chart]',
+})
+export class HighchartsChartDirective {
+  /**
+   * Highcharts library or Highcharts ESM module.
+   * @deprecated
+   */
+  Highcharts = input<Chart['highcharts']>();
+
+  /**
+   * Type of the chart constructor.
+   */
+  constructorType = input<string>('chart');
+
+  /**
+   * Callback function for the chart.
+   */
+  callbackFunction = input<Highcharts.ChartCallbackFunction>(null);
+
+  /**
+   * Whether to update the chart one-to-one.
+   */
+  oneToOne = input<boolean>();
```

Comment:
Could you please add this API doc here: https://api.highcharts.com/class-reference/Highcharts.Chart#update
With sth like:  Check how `update` works in Highcahrt

Reply:
@karolkolodziej , how about now ?